### ISSUE TITLE
dependency conflict with meteor 1.12

### ISSUE THREAD

**Issue by MastroLindus** [Meteor-Community-Packages/meteor-typescript#4]
Error while trying to add this package to a meteor 1.12 setup:

`meteor add refapp:meteor-typescript
 => Errors while adding packages:             
                                              
While selecting package versions:
error: Conflict: Constraint typescript@3.7.5 is not satisfied by typescript 4.1.2.
Constraints on package "typescript":
* typescript@~4.1.2 <- top level
* typescript@3.7.5 <- refapp:meteor-typescript-compiler 0.3.1 <- refapp:meteor-typescript 0.3.1`

**Comment by perbergland** [comment]
Ah OK yeah Typescript doesn’t use semantic versioning so the dependency check fails. Need to investigate how to relax this…

**Comment by perbergland** [comment]
This is the fix, thx to Jordan Baker for the sample code
```
  api.use(["typescript@3.0.0||4.0.0"], "server"); // For compiling this package
```

**Comment by perbergland** [comment]
Fixed in 0.3.2, just published
